### PR TITLE
feat(chronicle): Add ingestion labels support

### PR DESCRIPTION
**Pull request by Mrod1598**
### Proposed Change
- Added ingestion labels support
- Added Additional debug logging

##### Checklist
- [ ] Changes are tested
- [ ] CI has passed

### CODE REVIEW

File: exporter/chronicleexporter/README.md [hunk 1]
```diff
@@ -22,17 +22,18 @@ This exporter facilitates the sending of logs to Chronicle, which is a security
...
+| `raw_log_field`     | string            |                                                | `false`  | The field name for raw logs.                                                                |
+| `customer_id`       | string            |                                                | `false`  | The customer ID used for sending logs.                                                      |
+| `override_log_type` | bool              | `true`                                         | `false`  | Whether or not to override the `log_type` in the config with `attributes["log_type"]`       |
+| `namespace`         | string            |                                                | `false`  | User-configured environment namespace to identify the data domain the logs originated from. |
+| `compression`       | string            | `none`                                         | `false`  | The compression type to use when sending logs. valid values are `none` and `gzip`           |
+| `ingestion_labels`  | map[string]string |                                                | `false`  | Key-value pairs of labels to be applied to the logs when sent to chronicle.                 |
```
Comment (cpheps) [inline exporter/chronicleexporter/README.md:36]: Should we have an example showing this value used?

File: exporter/chronicleexporter/config.go [hunk 2]
```diff
@@ -65,6 +65,9 @@ type Config struct {
 
 	// Compression is the compression type that will be used to send logs to Chronicle.
 	Compression string `mapstructure:"compression"`
+
+	// Compression is the compression type that will be used to send logs to Chronicle.
```
Comment (cpheps) [inline exporter/chronicleexporter/config.go:69]: comment copy paste error

File: exporter/chronicleexporter/exporter.go [hunk 3]
```diff
@@ -113,6 +124,7 @@ func (ce *chronicleExporter) logsDataPusher(ctx context.Context, ld plog.Logs) e
 			continue
 		}
 
+		ce.logger.Debug("Sending payload to Chronicle", zap.ByteString("payload", data))
```
Comment (cpheps) [inline exporter/chronicleexporter/exporter.go:127]: Should we be logging this? Could there be sensitive data in the payload?

File: exporter/chronicleexporter/exporter.go [hunk 4]
```diff
@@ -165,5 +177,6 @@ func (ce *chronicleExporter) uploadToChronicle(ctx context.Context, data []byte)
 		return fmt.Errorf("received non-OK response from Chronicle: %s", resp.Status)
 	}
 
+	ce.logger.Debug("Received response from Chronicle", zap.String("status", resp.Status), zap.ByteString("response", respBody))
```
Comment (cpheps) [inline exporter/chronicleexporter/exporter.go:180]: Same here is it safe to log this?

**Review by cpheps — CHANGES_REQUESTED** [review]

File: exporter/chronicleexporter/README.md (hunk 1 above)
Comment (Mrod1598) [reply exporter/chronicleexporter/README.md:36]: Added a new example

File: exporter/chronicleexporter/config.go (hunk 2 above)
Comment (Mrod1598) [reply exporter/chronicleexporter/config.go]: fixed

File: exporter/chronicleexporter/exporter.go (hunk 3 above)
Comment (Mrod1598) [reply exporter/chronicleexporter/exporter.go]: yeah now that I think about it it does have the customer id which you can't really do anything with on its own but probably still not great. removed.

**Review by cpheps — APPROVED** [review]

**Review by BinaryFissionGames — APPROVED** [review]

**Pull request merged**